tfjs-node/README.md

```diff
@@ -66,6 +66,49 @@ To use this package on Raspberry Pi, you need to rebuild the node native addon w
 $ npm rebuild @tensorflow/tfjs-node --build-from-source
 ```
 
+#### Custom binaries URI
+
+If you happen to be using a mirror for the libtensorflow binaries (default is [https://storage.googleapis.com/]), you have 3 options (in order of priority):
+
+1. Set the environment variable `CDN_STORAGE`
```

Comment:
should this variable also support npm config file?

Reply:
I thought that it might be best to deprecate `CDN_STORAGE` at some major revision point and discontinue use in favor of the prefix versions: `TFJS_NODE_CDN_STORAGE` and `npm_config_TFJS_NODE_CDN_STORAGE`. As such, I didn't want to introduce the npm config file version of `CDN_STORAGE` as it would just get deprecated.

Please let me know how you'd like to proceed.